Formula/b/btop.rb

```diff
@@ -17,19 +17,29 @@ class Btop < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "af5c08bf52c9d84b54b3fe3992240a50c048ff3b897e8ee13f428fac4db76e0f"
   end
 
+  depends_on "coreutils" => :build
+
+  # Requires Clang 16.0.0, GCC 10.1.0
   on_macos do
-    depends_on "coreutils" => :build
-    depends_on "gcc"
+    depends_on "llvm" => :build if DevelopmentTools.clang_build_version < 1600
   end
 
-  fails_with :clang # -ftree-loop-vectorize -flto=12 -s
+  fails_with :clang do
+    build 1300
```

Comment:
What is the gap in Clang builds here for?
- Apple Clang 1300 is based on LLVM 12 and last for macOS 11
- Apple Clang 1600 is not released (latest is Apple Clang 1500 based on LLVM 16 which is available for macOS 13.5+)
- So above logic allows Apple Clang but also installs LLVM Clang for macOS 12, 13, and 14.

Reply:
just saw the comment `# Requires Clang 16.0.0, GCC 10.1.0`